Example/Source/Controllers/OptionsController/OptionsView.swift

```diff
@@ -25,25 +25,25 @@ class OptionsView: UIView {
     private let internalCarouselDemoButton = FilledButton()
     private let logoImageView = UIImageView()
     private let label = UILabel()
-    
+
     init() {
         super.init(frame: CGRect.zero)
         backgroundColor = UIColor.white
```

Comment:
It would be nice to have rule for type inference, but unfortunately I haven't found any in [Rules.md](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md)